### DISCUSSION TITLE
Tailwind CLI not generate CSS base on base css file

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tailwindlabs/tailwindcss#6930] [Help] [answered]
**What version of Tailwind CSS are you using?**

3.0.11

**What build tool (or framework if it abstracts the build tool) are you using?**

not use

**What version of Node.js are you using?**

14

**What browser are you using?**

Chrome
**What operating system are you using?**

Windows
**Reproduction URL**

**Describe your issue**

Hi, I am trying create css file from a base css file
```
/* @tailwind base; */
/* Here is its content */
@tailwind components;
@tailwind utilities;
```
Here is my config file
```
module.exports = {
  prefix: 'btb-',
  content: ['.src/**/*.js', '.src/**/*.ts', '.src/**/*.tsx'],
  // darkMode: false, // or 'media' or 'class'
  theme: {
    colors: {
     // just some custom color here
    },
    fontSize: {
// custom font
    },
    extend: {
      // custom
    },
  },
  variants: {
    extend: {},
  },
  plugins: [
    require('tailwindcss-pseudo-elements'),
    require('@tailwindcss/aspect-ratio'),
    require('@tailwindcss/line-clamp'),
  ],
};


```

But after I run `npx tailwindcss -c ./tailwind.config.js -i ./base.css -o ./src/assets/css/btb.scss`

I got content in `btb.scss`
```
/* @tailwind base; */
/* Here is its content */
```
Just that, no more css, I tried to roll back to verson 2.x.x, everything work fine. Why :c

**Comment by adamwathan** [comment, marked as answer]
Hey! Your content configuration has invalid paths:

```
  content: ['.src/**/*.js', '.src/**/*.ts', '.src/**/*.tsx'],
```

You are missing the `/` before `src`:

```
  content: ['./src/**/*.js', './src/**/*.ts', './src/**/*.tsx'],
```
